client/components/AttributionEditor/AttributionDetailControls.tsx

```diff
@@ -105,28 +121,41 @@ const AttributionDetailControls = (props: Props) => {
 						}
 					}}
 					onBlur={(evt) =>
-						onAttributionUpdate({
+						onAttributionUpdateWithValidation({
 							id,
 							affiliation: evt.target.value.trim(),
 						})
 					}
 				/>
 				{isShadowAttribution && (
-					<InputGroup
+					<InputField
 						// @ts-expect-error ts-migrate(2322) FIXME: Type '"" | Element | undefined' is not assignable ... Remove this comment to see the full error message
 						rightElement={orcid && <Tag minimal>ORCID</Tag>}
 						placeholder="ORCID"
 						defaultValue={orcid ?? undefined}
-						onKeyDown={(evt) => {
-							if (evt.key === 'Enter') {
-								evt.currentTarget.blur();
+						onChange={(evt) => {
+							const input = evt.target.value;
+							const orcidInvalidity = isOrcidInvalid(input);
+							setInvalidOrcid(orcidInvalidity);
+
+							if (!orcidInvalidity) {
+								onAttributionUpdateWithValidation({
+									id,
+									orcid: input.trim(),
+								});
 							}
 						}}
```

Comment:
`onKeyDown` is not available for `InputField`, so I removed it. This is somewhat annoying bc this removes the ability to "save" on Enter for this field.

I can't seem to capture `Enter` events on `onChange`, so instead, if the value matches the RegEx, we just submit the change. I see no real harm in this, as we used to and still submit changes on blur.